packages/site-content/site/index.html

```diff
@@ -1,17 +1,8 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>webcomponents.org</title>
-    <style>
-      body {
-        margin: 0;
-      }
-    </style>
-  </head>
-  <body>
-    <wco-top-bar></wco-top-bar>
-    <h2>🚧 Under construction! 🚧</h2>
-    <script type="module" src="./js/homepage.js"></script>
-  </body>
-</html>
+---
+layout: "layouts/base.11ty.cjs"
+title: "Web Components"
+scripts:
+- "./js/homepage.js"
+---
+<wco-top-bar></wco-top-bar>
```

Comment:
So this file is still using nunjucks, but the result goes to the `content` parameter.

I guess I feel slightly fuzzy about why this page is a `html` file, but the base template is `js`. Is it possible for us to do `js` all the way down?

Reply:
Every layout/template can be in a different language in 11ty. We could make the content page in JS, but I think that HTML or markdown + Nunjucks is a pretty good experience for content pages. The layout is in JS instead of Nunjucks mainly to make it easy to call in the server-generated pages without running it through a template system.